build.gradle

```diff
@@ -0,0 +1,29 @@
+group 'com.waverleysoftware'
+version '0.0.1'
+
+apply plugin: 'java'
+
+sourceCompatibility = 1.8
+
+repositories {
+    jcenter()
+}
+
+ext.seleniumVersion = '2.41.0'
+
+dependencies {
+    compile('org.testng:testng:6.12')
+    compile group: 'org.seleniumhq.selenium', name: 'selenium-firefox-driver', version:seleniumVersion
```

Comment:
for proper selenium usage you need only `selenium-java` dependency... others are redundant